### PR TITLE
Add store cancel-delete command in pd-ctl

### PR DESCRIPTION
**Pull request by rleungx**
Signed-off-by: Ryan Leung <rleungx@gmail.com>

### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v6.1 (TiDB 6.1 versions)
- [ ] v6.0 (TiDB 6.0 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: pd-control.md [hunk 1]
```diff
@@ -1251,9 +1251,17 @@ Encoding 格式示例：
 >> store delete 1
 ```
 
+撤销使用 store delete 下线的 store，如撤销下线 store id 为 1 的 store：
```
Comment (nolouch) [inline pd-control.md:1254]: 需要说明一下 tombstone 是没法 cancel 的？或者只有处于 Offline 的状态才行？

Comment (rleungx) [reply pd-control.md:1254]: ok

File: pd-control.md [hunk 2]
```diff
@@ -1251,9 +1251,17 @@ Encoding 格式示例：
 >> store delete 1
 ```
 
+撤销使用 store delete 下线的处于 Offline 状态的 store，Tombstone 状态的 store 无法被撤销，如撤销下线 store id 为 1 的 store：
```
Comment (TomShawn) [inline pd-control.md:1254]: 撤销下线后，store 会再次上线吗？

**Review by nolouch — APPROVED** [review]

File: pd-control.md (hunk 2 above)
Comment (TomShawn) [reply pd-control.md:1254]:
```suggestion
撤销已使用 store delete 下线并处于 Offline 状态的 store。撤销后，该 store 会从 Offline 状态变为 Up 状态。注意，该命令无法使 Tombstone 状态的 store 变回 Up 状态。以下示例撤销已使用 store delete 下线的 store，其 store id 为 1：
```

File: pd-control.md [hunk 3]
```diff
@@ -1251,9 +1251,17 @@ Encoding 格式示例：
...
+
+```bash
+>> store cancel-delete 1
 ```
-......
-```
+
+> **注意：**
+>
+> 若下线过程中切换了 PD leader，需要手动修改 store limit
```
Comment (TomShawn) [inline pd-control.md:1264]:
```suggestion
> 若下线过程中切换了 PD leader，需要手动修改 store limit。
```

**Pull request merged**